pandas/tests/window/test_rolling_skew_kurt.py

```diff
@@ -25,6 +25,16 @@ def test_series(series, sp_func, roll_func):
     tm.assert_almost_equal(result.iloc[-1], compare_func(series[-50:]))
 
 
+@pytest.mark.parametrize("sp_func, roll_func", [["kurtosis", "kurt"], ["skew", "skew"]])
+def test_low_variance_series(low_variance_series, sp_func, roll_func):
+    sp_stats = pytest.importorskip("scipy.stats")
+
+    compare_func = partial(getattr(sp_stats, sp_func), bias=False)
+    result = getattr(low_variance_series.rolling(50), roll_func)()
+    assert isinstance(result, Series)
+    tm.assert_almost_equal(result.iloc[-1], compare_func(low_variance_series[-50:]))
```

Comment:
Please check the whole series.

```suggestion
    window = low_variance_series.rolling(50)
    result = getattr(window, roll_func)()
    expected = window.apply(compare_func)
    tm.assert_almost_equal(result, expected)
```